solr/solr-ref-guide/src/requesthandlers-and-searchcomponents-in-solrconfig.adoc

```diff
@@ -169,3 +169,15 @@ Many of the other useful components are described in sections of this Guide for
 * `TermVectorComponent`, described in the section <<the-term-vector-component.adoc#the-term-vector-component,The Term Vector Component>>.
 * `QueryElevationComponent`, described in the section <<the-query-elevation-component.adoc#the-query-elevation-component,The Query Elevation Component>>.
 * `TermsComponent`, described in the section <<the-terms-component.adoc#the-terms-component,The Terms Component>>.
+
+Other components that ship with Solr include:
+
+[cols="20,40,40",options="header"]
+|===
+|Component Name |Class Name |More Information
+|phrases |`solr.PhrasesIdentificationComponent` | Learn more in the {solr-javadocs}org/apache/solr/handler/component/PhrasesIdentificationComponent.html[PhrasesIdentificationComponent] java class.
```

Comment:
I understand the desire to avoid duplication here with the Javadoc content, but it might still be nice to have a 1 sentence description, so readers don't have to click through just to figure out whether they're interested.

That said, even without that, this PR raises the visibility of these components which is a Good Thing.  So I'm happy either way.

Reply:
Thanks @gerlowskija..   In looking at the page a day later, I think that a table that is more in the vein of `Component Name, Class Name (?), Purpose, More Information, where More Information is some links might be the way to go.   Or, dropping the Class Name as kind of an implementaiton detail that you can look up on the More Information page.   Which maybe gets me back to just one big list! I'll mull some more.

Reply:
Sounds good.  You've clearly put more thought into it than I have, so I'll be happy with whatever you land on.